### PR TITLE
fix: missing glyphs in chat history

### PR DESCRIPTION
**Pull request by duncanodhis**
I've made some changes to the `lib/screens/settings/chat_history.dart` file. The modifications include using the system font as a fallback to address issues with missing glyphs.

### CODE REVIEW

File: lib/screens/settings/chat_history.dart [hunk 1]
```diff
@@ -7,6 +7,7 @@ import 'package:rtchat/models/messages/twitch/emote.dart';
 import 'package:rtchat/models/messages/twitch/message.dart';
 import 'package:rtchat/models/messages/twitch/user.dart';
 import 'package:rtchat/models/style.dart';
+import 'dart:ui' as ui;
```
Comment (SputNikPlop) [inline lib/screens/settings/chat_history.dart:10]: This is not needed and may be removed.

**Review by SputNikPlop — CHANGES_REQUESTED** [review]

File: android/.settings/org.eclipse.buildship.core.prefs
Comment (SputNikPlop) [inline android/.settings/org.eclipse.buildship.core.prefs:1]: This file is personal and probably shouldn't be added to the project.

File: android/.project
Comment (SputNikPlop) [inline android/.project:1]: These changes are not needed.

File: pubspec.lock [hunk 2]
```diff
@@ -1123,5 +1123,5 @@ packages:
     source: hosted
     version: "1.0.0"
 sdks:
-  dart: ">=3.1.0 <4.0.0"
+  dart: ">=3.2.0-194.0.dev <4.0.0"
```
Comment (SputNikPlop) [inline pubspec.lock:1126]: Appears you are using either the beta or the main branch, may you switch to the stable branch of flutter

File: android/.project
Comment (SputNikPlop) [inline android/.project:1]: @duncanodhis this doesn't need to be deleted

**Review by SputNikPlop — CHANGES_REQUESTED** [review]

**Comment by kevmo314** [comment]
Looks good for Android. Please remove the eclipse file and it'll be good to merge. Thanks for the contribution!

**Pull request merged**